### PR TITLE
Cz trans

### PR DESCRIPTION
**Pull request by fvybiral**
Hi, here is full czech localization.

### CODE REVIEW

File: scripts/word-list.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<?php
```
Comment (fossecode) [inline scripts/word-list.php]: Hi, thanks for the PR. What is the purpose of this file?

File: config-examples/cz-config.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  "COUNTRY": "Česká Republika",
+  "COUNTRY_CODE": "cz",
+  "DB_PATH": "./covid_db",
+  "LOCALE": "cs-CZ",
+  "MAP_CENTER": "15.4749126, 49.8037633",
+  "MAP_MAX_ZOOM": 13,
+  "MAP_ZOOM": 7,
+  "PASSCODE_LENGTH": 3,
+  "REDIRECT_TO_GOVERNMENT": false,
+  "SUPPORTED_LOCALES": ["cz"],
```
Comment (fossecode) [inline config-examples/cz-config.json]: This should probably be `cs-CZ`

**Comment by sbocinec** [comment]
@fvybiral I've checked the wordlist, but IMO it contains not very suitable words - words in plural, not in a base form, similar one, etc. I suggest to use instead this bitcoin BIP-0039 https://github.com/bitcoin/bips/blob/master/bip-0039/czech.txt , what do you think?

**Comment by fossecode** [comment]
> @fvybiral I've checked the wordlist, but IMO it contains not very suitable words - words in plural, not in a base form, similar one, etc. I suggest to use instead this bitcoin BIP-0039 https://github.com/bitcoin/bips/blob/master/bip-0039/czech.txt , what do you think?

I agree. The bitcoin BIP-0039 should be very well suited for our needs.

File: config-examples/cz-config.json (hunk 2 above)
Comment (fvybiral) [reply config-examples/cz-config.json]: Changed

File: scripts/word-list.php (hunk 1 above)
Comment (fvybiral) [reply scripts/word-list.php]: Sry, it is temp file... We use php to convert one czech dictionary to plain text ;(

**Comment by fossecode** [comment]
I think you can delete `fileWithLotsOfWords.txt` as well?

File: package.json [hunk 3]
```diff
@@ -11,7 +11,8 @@
     "serve": "NODE_ENV=production ts-node app/server.ts",
     "dev:css": "./node_modules/.bin/tailwind build tailwind.config.css -c tailwind.config.js -o static/tailwind.min.css",
     "build:css": "postcss tailwind.config.css -o static/tailwind.min.css",
-    "build:images": "ts-node scripts/generate-images.ts"
+    "build:images": "ts-node scripts/generate-images.ts",
+    "make:wordlist": "php scripts/word-list.php"
```
Comment (fossecode) [inline package.json]: Should this be reverted as well? :)

**Comment by fvybiral** [comment]
As you wish..

File: README.md [hunk 4]
```diff
@@ -166,6 +167,10 @@ Download & install:
 
 Social images (social media share image, Twitter header and generic banner) can be generated by running `yarn build:images` while your dev server is running. They will be placed in the static folder, in the language you defined in your config.json (make sure `LOCALE`, `BASE_URL` and `COUNTRY_CODE` is set correctly). The social-image.png is then automatically linked as social media share image.
 
+### Generation localized word-list
```
Comment (fossecode) [inline README.md]: And this 🙈

Comment (fvybiral) [reply README.md]: Sry, Ondra is php programmer... Also is very good translater.

File: README.md [hunk 5]
```diff
@@ -38,6 +38,7 @@ We need people who can help translate the site, develop new features, project le
 - 🇹🇷 Turkey: https://coronastatustr.com
 - 🇱🇹 Lithuania: https://coronastatus.lt
 - 🇳🇵 Nepal: https://coronastatusnp.com
+- 🇨🇿 Czech Republic: http://corona-status.cz
```
Comment (fossecode) [inline README.md]: Final little detail, this should probably rather be `https`

File: README.md (hunk 4 above)
Comment (fossecode) [reply README.md]: 👍 no problem. The script was nice, and if we had had to generate the lists it would have been useful

**Review by fossecode — APPROVED** [review]
Awesome, great work guys

**Pull request merged**